src/Importify/Common.hs

```diff
@@ -155,3 +157,23 @@ importSlice []               = Nothing
 importSlice [ImportDecl{..}] = Just $ startAndEndLines importAnn
 importSlice (x:y:xs)         = Just $ startAndEndLines
                                     $ combSpanInfo (importAnn x) (importAnn $ NE.last (y :| xs))
+
+-- | Returns module name of the source file
+-- We can't parse the whole file to get it because default extensions are not available yet
+getModuleName :: Text -> String
+getModuleName src = case impl of
+                        [mn] -> L.init mn
+                        []   -> error "File doesn't have `module' declaration"
```

Comment:
Yes, that was one of the things I wanted to ask. We can't use a complete parser and language-haskell-exts doesn't come with a partial parser.